.github/workflows/test.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
     - name: Clippy Micro-RDK Installer
       run: |
-        bash -c 'make clippy-cli'
+        bash -c 'cargo update -p clap --precise v4.3.0 && make clippy-cli'
```

Comment:
?

Reply:
it just keeps failing on building clap even though I've made no changes to the way I use clap or to Cargo.toml. So this is me grasping at straws

Reply:
yeah, the thing is, it didn't happen in the previous flasher PR (which actually introduced using clap) and I can't replicate it locally. Essentially the complaint is that some clap dependency is requiring rustc 1.70, but again I have 1.67 locally and can't replicate the problem

Reply:
try to commit the Cargo.lock for the flasher

Reply:
I tried having the actions cargo clean in the directory and now it's trying to use clap v4.4 even though I've specifically marked "4.3" in the Cargo.toml